src/Access/UsersConfigAccessStorage.cpp

```diff
@@ -196,6 +196,14 @@ namespace
             user->access.revokeGrantOption(AccessType::ALL);
         }
 
+        /// Daisy : start.
+        const auto database_config = user_config + ".database";
```

Comment:
i think we shall avoid binding a user with a database at this level. At least not for now.